### PR TITLE
Cor 2290 - Webhooks Docs

### PR DESCRIPTION
**Pull request by katemacfarlane**
Webhooks Docs
-- Creating a Webhook
-- Example Payload
-- Testing a Webhook
-- Deleting a Webhook

### CODE REVIEW

File: docs/extras/webhooks/webhooks.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
+---
+title: Webhooks
+sidebar_position: 1
+---
+
+This topic explains how to create and use webhooks in DevCycle.
```
Comment (ajwootto) [inline docs/extras/webhooks/webhooks.md:6]: Webhooks should be capitalized everywhere I think

File: docs/extras/webhooks/webhooks.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+## Outbound Webhooks
+
+### Creating a Webhook
+
+To create a webhook:
+
+1. Navigate to the Integrations page.
+2. Navigate to the "Webhooks" section and click `+ New Webhook`. The "Create a Webhook" modal appears.
+3. Give the webhook a human-readable Name.
```
Comment (ajwootto) [inline docs/extras/webhooks/webhooks.md:19]:
```suggestion
3. Give the webhook a human-readable name.
```

File: docs/extras/webhooks/webhooks.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+4. (Optional) Give the webhook a description. 
+5. Enter a Payload URL.
+6. Click `Create Webhook`. From there, you will be taken to the Webhook details page. 
+7. Select if you'd like events sent for all Features in your project or a single Feature. 
+8. Select which events will be sent through the webhook. 
+9. Click `Save`. 
+
+### Example Payload 
+
+This is for cases of project level Webhooks where the Feature isn't necessarily implied.
```
Comment (ajwootto) [inline docs/extras/webhooks/webhooks.md:29]:
```suggestion
This is for cases of project-level Webhooks where the Feature isn't necessarily implied. 
```

File: docs/extras/webhooks/webhooks.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+6. Click `Create Webhook`. From there, you will be taken to the Webhook details page. 
+7. Select if you'd like events sent for all Features in your project or a single Feature. 
+8. Select which events will be sent through the webhook. 
+9. Click `Save`. 
+
+### Example Payload 
+
+This is for cases of project level Webhooks where the Feature isn't necessarily implied. 
+
+```json
```
Comment (ajwootto) [inline docs/extras/webhooks/webhooks.md:31]: this is Typescript not json

File: docs/extras/webhooks/webhooks.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+	user: User
+}
+]
+```
+
+For example, a user edits a Feature’s key and adds a new variable, the data posted to the user’s Webhook URL would be:
+
+```json
+{
+	events: ['modifiedFeature', 'addedVariable'],
```
Comment (ajwootto) [inline docs/extras/webhooks/webhooks.md:96]: valid JSON requires "quotes" around the properties

File: docs/extras/webhooks/webhooks.md [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+To delete a webhook:
+
+1. Navigate to the Integrations page.
+2. Navigate to the "Webhooks" section.
+3. Click on the Webhook that you wish to delete.
+4. Navigate to the Settings section of the Webhook details page. 
+5. Click `Delete Webhook`. A confirmation modal will appear.
+6. Click `Delete`.
+
+## Incoming Webhooks (Coming Soon)
```
Comment (ajwootto) [inline docs/extras/webhooks/webhooks.md:147]: should this include some message about what these will be and how to get in contact if you're interested?

**Review by jsalaber — APPROVED** [review]

**Pull request merged**